src/js/view/controls/rightclick.js

```diff
@@ -84,13 +83,19 @@ export default class RightClick {
         return false;
     }
 
-    hideMenu() {
-        this.elementUI.off('out', this.hideMenu, this);
-        if (this.mouseOverContext) {
+    hideMenu(evt) {
+        if (!this.model.get('touchMode')) {
             // If mouse is over the menu, hide the menu when mouse moves out
-            this.elementUI.on('out', this.hideMenu, this);
+            this.el.removeEventListener('mouseout', this.hideMenuHandler);
+            if (this.mouseOverContext) {
+                this.el.addEventListener('mouseout', this.hideMenuHandler);
```

Comment:
Can this be added once in lazySetup instead of on every `hideMenu` call?

Reply:
Looks like the reason this is here is that after you right click, you're technically inside the menu, so as soon as you move the mouse, the menu will hide.

Reply:
Right. So then what's wrong with always having this listener?

Reply:
Yes

Reply:
@hghazzi can we close the loop on this? Not sure what's meant by 'yes'.